packages/lesswrong/components/votes/ReactionsPalette.tsx

```diff
@@ -144,10 +165,11 @@ const ReactionsPalette = ({getCurrentUserReaction, getCurrentUserReactionVote, t
       <div>
         <span>{reaction.label}</span>
         <ReactionDescription reaction={reaction} classes={classes}/>
-        {reaction.deprecated && "This react has been deprecated and may be removed later"}
+        {/*{reaction.deprecated && "This react has been deprecated and may be removed later"}*/}
```

Comment:
probably either get rid of this completely, or, leave it fully in because if somehow a deprecated react *does* appear we probably want to actually warn users about this.